geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/coprocessor/GeoMesaCoprocessor.scala

```diff
@@ -94,14 +94,13 @@ class GeoMesaCoprocessor extends GeoMesaCoprocessorService with Coprocessor with
           scanner.close()
         }
       }
-
-      GeoMesaCoprocessorResponse.newBuilder.addAllPayload(results.map(ByteString.copyFrom).asJava).build
     } catch {
-      case e: IOException => ResponseConverter.setControllerException(controller, e); null
-      case NonFatal(e) => ResponseConverter.setControllerException(controller, new IOException(e)); null
+      case _: InterruptedException | _ : InterruptedIOException => // stop processing, but don't return an error to prevent retries
```

Comment:
Are there situations where we ought to retry?!

Reply:
I think if we get an interrupted exception then we don't want to retry... if we get a non-interrupted exception then we return that and it causes hbase to retry the call. At least, that's the whole basis for this PR :)

@malinsinbigler do you have time to test out the interrupt with this branch?

Reply:
Sure thing, I'll test the 2.4.0 co-processor

Reply:
@elahrvivaz running `./geomesa-hbase stats-count` does not time out as expected.